plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -289,54 +289,56 @@ public void testCreateRefreshSelect()
         MaterializedResult baseResult = computeActual("SELECT * from materialized_view_no_part");
         assertEquals(baseResult.getRowCount(), 6);
         String plan = getExplainPlan("SELECT * from materialized_view_no_part", ExplainType.Type.IO);
-        assertTrue(plan.contains("base_table1"));
+        assertThat(plan).contains("base_table1");
         assertUpdate("REFRESH MATERIALIZED VIEW materialized_view_no_part", 6);
         MaterializedResult viewResult = computeActual("SELECT * from materialized_view_no_part");
         assertEquals(viewResult.getRowCount(), 6);
         plan = getExplainPlan("SELECT * from materialized_view_no_part", ExplainType.Type.IO);
-        assertFalse(plan.contains("base_table1"));
+        assertThat(plan).doesNotContain("base_table1");
 
         baseResult = computeActual("SELECT * from materialized_view_agg");
         assertEquals(baseResult.getRowCount(), 3);
         plan = getExplainPlan("SELECT * from materialized_view_agg", ExplainType.Type.IO);
-        assertTrue(plan.contains("base_table1"));
+        assertThat(plan).contains("base_table1");
         assertUpdate("REFRESH MATERIALIZED VIEW materialized_view_agg", 3);
         viewResult = computeActual("SELECT * from materialized_view_agg");
         assertEquals(viewResult.getRowCount(), 3);
         plan = getExplainPlan("SELECT * from materialized_view_agg", ExplainType.Type.IO);
-        assertFalse(plan.contains("base_table1"));
+        assertThat(plan).doesNotContain("base_table1");
         assertQuery(session, "SELECT * from materialized_view_agg", "VALUES (DATE '2019-09-10', 2)," +
                 "(DATE '2019-09-08', 1), (DATE '2019-09-09', 3)");
 
         baseResult = computeActual("SELECT * from materialized_view_part");
         assertEquals(baseResult.getRowCount(), 3);
         plan = getExplainPlan("SELECT * from materialized_view_part", ExplainType.Type.IO);
-        assertTrue(plan.contains("base_table1"));
+        assertThat(plan).contains("base_table1");
         assertUpdate("REFRESH MATERIALIZED VIEW materialized_view_part", 3);
         viewResult = computeActual("SELECT * from materialized_view_part");
         assertEquals(viewResult.getRowCount(), 3);
         plan = getExplainPlan("SELECT * from materialized_view_part", ExplainType.Type.IO);
-        assertFalse(plan.contains("base_table1"));
+        assertThat(plan).doesNotContain("base_table1");
 
         baseResult = computeActual("SELECT * from materialized_view_join");
         assertEquals(baseResult.getRowCount(), 5);
         plan = getExplainPlan("SELECT * from materialized_view_join", ExplainType.Type.IO);
-        assertTrue(plan.contains("base_table1") && plan.contains("base_table2"));
+        assertThat(plan).contains("base_table1").contains("base_table2");
```

Comment:
WDYT? Here and other places.
```suggestion
        assertThat(plan).contains("base_table1", "base_table2");
```

Reply:
makes sense, updated